tests/core/full_node/test_full_node.py

```diff
@@ -665,7 +666,7 @@ async def test_request_sub_blocks(self, wallet_nodes):
         )
         assert len(res.data.sub_blocks) == 6
         assert res.data.sub_blocks[-1].transactions_generator is not None
-        assert res.data.sub_blocks[-1] == blocks[-1]
+        assert std_hash(res.data.sub_blocks[-1]) == std_hash(blocks_t[-1])
```

Comment:
do subblocks not come with comparison operators?

Reply:
nope

Reply:
We should use .header_hash or .get_hash() instead of std_hash though